Home page text alignment

jjophi opened the pull request:
Changed position and alignment of homepage news link.

jjophi committed:
Home page text alignment

manuignatius reviewed (changes requested):
Overall looks good, 3 comments.
1. We had a small orange underline below the tagline, its gone now, was that intentional?
2. Instead of "In the News", can we have another "term" what is similar to "Whats New"? As this is about something interesting from Subnero and may not always be news-like.
3. Can you have another version where there is more spacing from the tagline (move the "new" section further below)?

For future ones, if you can add a screenshot of the change in the PR, that would also help.

jjophi committed:
Home page latest news

jjophi commented:
Updated the page with these changes:
1. Underline below the tag line added back.
2. 'In the news' text changed to 'What's new?'
3. Spacing between tagline and news section is increased.

Also updated the news section to show latest three posts but showing one at a time with a transition.

![127 0 0 1_4000_](https://github.com/subnero1/subnero1.github.io/assets/20460360/86c55e1b-2d89-4c9f-b0ea-7e0fc995676d)

jjophi committed:
Home page news section placement

jjophi commented:
Updated the 'What's new?' section - moved it to bottom of the page.
![127 0 0 1_4000_ (1)](https://github.com/subnero1/subnero1.github.io/assets/20460360/1cd73a32-e6d0-4e91-964b-5ae15993f59d)
